src/components/BottomNav.js

```diff
@@ -26,12 +24,11 @@ export default function FullWidthTabs() {
 
   return (
     <div className={classes.root}>
-      <AppBar position="static" color="default">
+      <AppBar position="static" color="primary">
         <Tabs
           value={value}
           onChange={handleChange}
           indicatorColor="primary"
```

Comment:
If we keep `primary` color as black maybe this should be the `secondary` color, though I'm not sure if the contrast will be strong enough

Reply:
But right now the top navbar is black/primary, so wouldn't it look good if they matched?

Reply:
Oh no! So sorry, I should've tagged this with the `Tabs` component!